testing/test_aux.sh

```diff
@@ -191,4 +195,65 @@ head inst/run-port-*/finish/nmap*/*
 
 tcpdump -en -r inst/run-port-01/scans/test_nmap.pcap icmp or arp
 
+
+function monitor_interface {
+    while true; do 
+        found=$(cat inst/cmdrun.log 2>/dev/null | grep "$2")
+        if [ -n "$found" ]; then
+            echo found $2
+            eval $1 
+            break
+        fi
+        test_done=$(cat $TEST_RESULTS | grep "Done with tests")
+        if [ -n "$test_done" ]; then
+            break
+        fi
+    done &
+}
+rm -f inst/cmdrun.log
+# Check port toggling does not cause a shutdown 
+cp misc/system_base.conf local/system.conf
+echo "port_toggle_timeout_sec=10" >> local/system.conf
+echo "port_debounce_sec=0" >> local/system.conf
+monitor_interface "sudo ifconfig faux down;sleep 1; sudo ifconfig faux up" "Port 1 dpid 2 is now active"
+cmd/run -s  
```

Comment:
Is there a way to combine these 3x cases into one + unit_test coverage?  Seems like there's only really two behavioral cases that need to be captured -- flap is fast (no problem), or flap is too slow (errors out) -- everything else (different settings of values) should be accomplished with a unit test (so no need to run cmd/run).  Anurag added the unit test framework in, so it should be easier to use that.

I'm trying to stem the testing bloat that will come from too many integration tests for everything. They're "easy" -- but also really slow and add a lot of technical overhead to the system in the long run.